projects/amd-loader/src/loader/config.js

```diff
@@ -207,6 +207,22 @@ export default class Config {
 		return dependencyModule;
 	}
 
+	/**
+	 * Update the resolve path (usually as a consequence of a redirect being
+	 * received from the server when a module resolution is attempted).
+	 * @param {string} resolvePath the updated resolve path
+	 * @return {void}
+	 */
+	updateResolvePath(resolvePath) {
+		const prefix = `${window.location.protocol}//${window.location.host}`;
```

Comment:
nit: I think you can also use `new URL` so that you don't have to manually concat. Although if this is for a fie path, not sure itd work as expected

Reply:
:thinking: I don't think so, because [URL's constructor](https://developer.mozilla.org/en-US/docs/Web/API/URL/URL) would need to get ``${window.location.protocol}//${window.location.host}`` in any case...

Or am I missing anything?

Reply:
For the reference: this is just an optimization to remove the host and port from the redirect URL because, when configured from scratch, the AMD loader is missing them too. But if we didn't remove it, it would work too, so we don't need to be exhaustive when `cleaning` the URL.

Reply:
By exhaustive I mean that this replacement would fail if the URL had credentials in it, for example.

Reply:
ah I gotcha, I think I was actually mistaken with URL, my mind was somewhere else...

Reply:
Maybe you were thinking in Java's URL class... :monocle_face: